packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -206,28 +264,65 @@ function HeaderMenu( { dataView, header } ) {
 										} }
 									>
 										{ element.label }
-									</DropdownMenuItemV2>
+									</DropdownMenuItem>
 								);
 							} ) }
-						</DropdownSubMenuV2>
-					</DropdownMenuGroupV2>
+						</DropdownMenu>
+					</DropdownMenuGroup>
 				) }
 			</WithSeparators>
-		</DropdownMenuV2>
+		</DropdownMenu>
 	);
-}
+} );
 
 function WithSeparators( { children } ) {
 	return Children.toArray( children )
 		.filter( Boolean )
 		.map( ( child, i ) => (
 			<Fragment key={ i }>
-				{ i > 0 && <DropdownMenuSeparatorV2 /> }
+				{ i > 0 && <DropdownMenuSeparator /> }
 				{ child }
 			</Fragment>
 		) );
 }
 
+const Cell = forwardRef( function Cell( { cell, ...props }, ref ) {
+	const [ widgetUsed, setWidgetUsed ] = useState( false );
+
+	const Widget = forwardRef( ( widgetProps, innerRef ) => {
+		useEffect( () => setWidgetUsed( true ), [] );
+		return <CompositeItem ref={ innerRef } { ...widgetProps } />;
+	} );
```

Comment:
This is the bit that we pass through to individual cell content render functions. If they make use of it, we adjust the layout here to not make the cell itself navigable.